command/ca/provisioner/add.go

```diff
@@ -15,33 +15,48 @@ import (
 	"github.com/urfave/cli"
 )
 
-const (
-	jwkType  = "JWK"
-	oidcType = "OIDC"
-)
-
 func addCommand() cli.Command {
 	return cli.Command{
 		Name:   "add",
 		Action: cli.ActionFunc(addAction),
 		Usage:  "add one or more provisioners the CA configuration",
 		UsageText: `**step ca provisioner add** <name> <jwk-file> [<jwk-file> ...]
-		[**--ca-config**=<file>] [**--create**] [**--password-file**=<file>]`,
+**--ca-config**=<file> [**--type**=JWK]  [**--create**] [**--password-file**=<file>]
```

Comment:
shouldn't this go in examples instead of usage text?

Reply:
This method is quite complicated, so I wanted to make make the usage more verbose, with the main different methods to use it

It looks something like:
```
USAGE
      step ca provisioner add name jwk-file [jwk-file ...] --ca-config=file
      [--type=JWK] [--create] [--password-file=file]

      step ca provisioner add name --type=OIDC --ca-config=file [--client-id=id]
      [--client-secret=secret] [--configuration-endpoint=url] [--domain=domain]
      [--admin=email]...

      step ca provisioner add --type=[AWS|Azure|GCP] --ca-config=file
      [--aws-account=id] [--gcp-service-account=name] [--gcp-project=name]
      [--azure-tenant=id] [--azure-resource-group=name] [--instance-age=duration]
      [--disable-custom-sans] [--disable-trust-on-first-use]
```